### PR TITLE
Simplify README by making it point to RTD.

### PR DESCRIPTION
**Pull request by ijnek**
Documentation has been ported to RTD.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,233 +4,6 @@
 
 Package that contains libraries to display [soccer_vision_3d_msgs](https://index.ros.org/p/soccer_vision_3d_msgs/) as RViz Markers.
 
-The node simply listens for ball, field boundary, goalposts, markings, obstacles and robots,
-converts them to visualization markers and publishes them so they can be visualized in RViz.
+This package is only available currently for ROS2 Rolling.
```
Comment (jaagut) [inline README.md]:
```suggestion
This package is currently only available for ROS2 Rolling.
```
For me, this feels more correct, but only I speak English second as language 😉

File: README.md [hunk 2]
```diff
@@ -4,233 +4,6 @@
...
-      size:
-        x: 0.2
-        y: 0.2
-        z: 0.4
-    attributes:
-      player_number: 3
-      team: 2
-"
-```
+For more information, see our [Documentation](https://soccer-vision-3d-rviz-markers.readthedocs.io/)
```
Comment (jaagut) [inline README.md]:
```suggestion
For more information, see our [Documentation](https://soccer-vision-3d-rviz-markers.readthedocs.io/).
```

**Review by jaagut — APPROVED** [review]

**Comment by ijnek** [comment]
@jaagut thanks for the review and the suggestions!

File: README.md (hunk 1 above)
Comment (jaagut) [reply README.md]: Just to clarify, I didn't know if I was right or wrong with my suggestion, it was just a feeling.
The comment about English being my second language was just supposed to mean, that I was not sure. It was in no way meant as a comparison or affront.

I felt the need to clarify, as I noticed, I didn't really convey the reason for my last comment:

> For me, this feels more correct, **but only I speak English second as language wink**

Comment (ijnek) [reply README.md]: Both are correct, but your suggestion is more natural!

**Pull request merged**